cmd/query/app/server.go

```diff
@@ -203,14 +203,13 @@ type httpServer struct {
 
 var _ io.Closer = (*httpServer)(nil)
 
-func createHTTPServer(
-	ctx context.Context,
+func initRouter(
```

Comment:
will it be okay to have redundant handlers with the otel implementation? specifically asking about the response headers handler and the compression handler.

Reply:
yes